### DISCUSSION TITLE
oVirt Engine Active Directory Authenrtication

### DISCUSSION THREAD

**Discussion by mikejhathaway** [oVirt/ovirt-site#3124] [Q&A] [answered]
Hi Folks,

I finally got my engine installed and it seems to be running ok, I am trying to setup active directory authentication for users, I am following the steps in the oVirt docs, but after I restart the engine service, the profile drop down does not appear on the login page, I have rebooted the server and cleared the browser cache as well as trying incognito mode and other browsers. 

https://www.ovirt.org/documentation/administration_guide/index.html#Attaching_an_Active_Directory

Has anyone else experienced this with oVirt 4.5?

TIA 

Mike

**Comment by dupondje** [comment, marked as answer]
The ovirt-engine-extension-aaa-ldap package is deprecated and AD auth can be done via Keycloak which is the preferred way.
